__tests__/app.test.js

```diff
@@ -163,3 +163,50 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: should respond with an object with comment_count property", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(res => {
+        expect(res.body.article).toEqual(
+          expect.objectContaining({
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            title: expect.any(String),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(String),
+          })
+        );
+        expect(res.body.article.comment_count).toBe("11");
+      });
+  });
+  test("200: should respond with correct number of comments for the given article id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(res => {
+        expect(res.body.article.comment_count).toBe("11");
+      });
+  });
+  test("200: should respond with correct number of comments for the given article id", () => {
```

Comment:
What do we gain by repeating this test three times?

Reply:
OK Right.
I will remove the unnecessary tests